### PR TITLE
New install file

### PR DESCRIPTION
**Pull request by Hellebore**

### CODE REVIEW

File: welcome-to-sourcery.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# Welcome to Sourcery! Once you've added the token, look for the
+# underlined `result = []` below. This is a suggestion from Sourcery.
+# Hover over it to see details of the changes including a diff. 
+
+def refactoring_example(spellbook):
+    result = []
+    for spell in spellbook:
+        if spell.is_awesome:
+          result.append(spell)
```
Comment (brendanator) [inline welcome-to-sourcery.py:18]: Add two spaces to format it the same as other lines

File: welcome-to-sourcery.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    return result
+
+# Apply the suggestion by taking the quick-fix action:
+#   Put the cursor on the line and either click the lightbulb or use the hotkey:
+#   (Ctrl + .) or (Cmd + .) by default. Take the 'Convert for loop...' option.
```
Comment (brendanator) [inline welcome-to-sourcery.py:23]: The style of these keyboard shortcuts is different from the line before.

File: welcome-to-sourcery.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+def refactoring_example(spellbook):
+    result = []
+    for spell in spellbook:
+        if spell.is_awesome:
+          result.append(spell)
+    return result
+
+# Apply the suggestion by taking the quick-fix action:
+#   Put the cursor on the line and either click the lightbulb or use the hotkey:
```
Comment (brendanator) [inline welcome-to-sourcery.py:22]: Maybe 'highlighted line'

**Review by brendanator — CHANGES_REQUESTED** [review]
Looks really good. Just a couple of small things

File: welcome-to-sourcery.py (hunk 1 above)
Comment (Hellebore) [reply welcome-to-sourcery.py:18]: done

File: welcome-to-sourcery.py (hunk 3 above)
Comment (Hellebore) [reply welcome-to-sourcery.py:22]: done

File: welcome-to-sourcery.py (hunk 2 above)
Comment (Hellebore) [reply welcome-to-sourcery.py:23]: reworded

**Pull request merged**